src/components/dialogs/inputs/input-hooks.js

```diff
@@ -18,8 +18,11 @@ import { validateField } from '../../util/validation-functions';
 import {
     CircularProgress,
     FormHelperText,
+    FormLabel,
     InputLabel,
     MenuItem,
+    Radio,
+    RadioGroup,
```

Comment:
maybe move the import of `FormControlLabel` here as well ?

Reply:
Yes, thank you, now I'll check on this kind of stuff after using the auto include function of my IDE.